### ISSUE TITLE
Missing UI controls

### ISSUE THREAD

**Issue by thijs** [ChristopherHX/mcpelauncher-manifest#55]
I'm trying out this version of the launcher as the one I've been using can't run anything higher than 1.14.1.
It looks great, except there are a few options and controls missing:

- unable to import worlds: in the other launcher I have an import button next to the 'create world' button on the worlds tab.
- in the 'create world' menu it is missing entries in 'my world templates': the other launcher correctly displays maps I've bought from the store here to start playing.

**Comment by ChristopherHX** [comment]
import world menu button isn't available on android
(patch removed, you can do it by hand like on android)
Download your templates again, this version patches the game not 1:1 so the location of the cache are not equal

**Comment by ChristopherHX** [comment]
Might be added by a seperate mod. The launcher contains a me unknown modding api

**Comment by ChristopherHX** [comment]
Look at this WIP, mod to get some patches back
It recovers the missing (not really, its a win10 exclusive) Button.
win10 ui on old releases, symbol errors, different scaling rules and msaa option
https://github.com/ChristopherHX/mcpelauncherlegacy/releases

**Comment by thijs** [comment]
I'm afraid I'll need a little more information to try that out.
Where do I put the two lib files? Should I add the mod directory into the appimage somehow, or add the dir in .local/share/mcpelauncher somewhere? I tried that last, but that doesn't do anything...

**Comment by ChristopherHX** [comment]
for linux inside .local/share/mcpelauncher/mods/*.so
gnustlshared.so and the mod.so

**Comment by thijs** [comment]
Yeah. That brings back the button, but as you say the file picker doesn't show so it fails. But it's a good step in the right direction, it looks like.

**Comment by ChristopherHX** [comment]
Information, Version 0.0.1 of the mod will crash with the release of ng (minecraft-linux's merge of my forks approach)
Please update to 0.0.3 or newer to fix the potential crash
